### DISCUSSION TITLE
Notifications, Plugins and Releases

### DISCUSSION THREAD

**Discussion by ilovethickthighs** [japandotorg/LemonCord#16] [Ideas]
I was wondering if it's possible to implement those features.

Notifications are pretty basic need for discord application, and i don't think i have to explain why i would need them.

Plugins, Is there a way to inject plugin managers like vencord or shelter, or maybe even better discord plugins?

Releases, I managed to install lemoncord but it would be nice having a gui installer soon.

These are not demands, I would just love to see those features being implemented on lemoncord, If not it's all good

**Comment by japandotorg** [comment]
I do plan on implementing such things in the future, however there's no ETA on when I'll be able to do it because of all the work pressure I have been getting into lately.

**Reply by ilovethickthighs** [reply]
its fine, I understand, take your time. No pressure
